main.cpp

```diff
@@ -13,75 +13,506 @@ What you create in this project will be used as the basis of Project 5 in the co
+
+        void performSongLive(bool tour = true);
+        void recordSongInTheStudio(bool recordAlbum);
+        void promoteSong(int numberOfInterviews, bool tvAppearances);
+    };
+
+    Artist artistA;
+
+    // duration of the song
+    int duration = 5;
+    // tempo of the song
+    int tempo = 125;
+    // key of the song
+    char key = 'a';
+    // number of artists performing on the song
+    int numOfPerformingArtists = 4;
+    // record label that released the song
+    std::string recordLabel = "Virgin";
+    
+    // play song on the radio
+    void playSongOnRadio(int duration, int tempo);
+    // perform song live at a concert
+    void performSongLiveAtConcert(Artist artistB);
+    // remix the song
+    void remix(std::string title, Artist artistC, std::string recordLabel);
+};
+
+/*
+Thing 2) Sandwich
+5 properties:
+    1) type of bread // string
+    2) type of fillings // string
+    3) amount of different fillings // int
+    4) condiment used // string
+    5) number of calories // int
+3 things it can do:
+    1) cause foodpoisoning
+    2) provide a satisfying meal
+    3) go moldy if left uneaten
+ */
+
+struct Sandwich
+{
+    // type of bread
+    std::string bread = "brown";
+    // type of fillings
+    std::string fillings = "meat";
+    // amount of different fillings
+    int numOfFillings = 2;
+    // condiment used
+    std::string condiment = "mayo";
+    // number of calories
+    int calories = 600;
+
+    // cause foodpoisoning
+    void foodPoisoning(std::string fillings = "uncooked chicken");
+    // provide a satisfying meal
+    void satisfyingMeal(int calories);
+    // go moldy if left uneaten
+    void moldy(int daysWithoutStorage);
+};
+
+/*
+Thing 3) Soccer Team
+5 properties:
+    1) number of players in squad // int
+    2) average age of players in squad // int
+    3) number of total goals scored by the team // int
+    4) number of matches in the season // int
+    5) league position // int
+3 things it can do:
+    1) train their skills
+    2) win a game
+    3) move up the league table
+ */
+
+struct SoccerTeam
+{
+    class Manager
+    {
+        std::string name = "Pep Gaurdiola";
+        int Age = 48;
+        std::string birthPlace = "Spain";
+        int yearsActive = 15;
+        int numberOfTeamsManaged = 3;
+
+        void trainTeam(std::string nameOfCoach, bool deligateTrainingSession);
+        void signNewPlayers(std::string market, int age, std::string playerPosition);
+        void speakToThePress(bool attendPressConference = true);
+    };
+
+    // number of players in squad
+    int squadPlayers = 24;
+    // average age of players in squad
+    int averageAge = 28;
+    // number of total goals scored by the team
+    int totalGoalsScored = 60;
+    // number of matches in the season
+    int numOfMatchesToPlay = 38;
+    // league position
+    int leaguePosition = 4;
+
+    // train their skills
+    void trainingSession(Manager managerA);
```

Comment:
This names contain the 'action' or verb.
change your function name to match the plain-english description
- [ ] revise